src/EventTooltip.js

```diff
@@ -163,9 +161,18 @@ const TooltipReactEvent = ({
     case 'schedule-state-update':
       label = 'state update scheduled';
       break;
-    case 'suspend':
+    case 'schedule-force-update':
```

Comment:
This is very much a code style suggestion and out of scope for this PR, but I much prefer these types of mapping switches to be extracted to their own functions (`getEventTypeLabelText(type` etc), instead of mutating a local variable.

Reply:
Filed as #37